teamworkApp/lib/studentGraph.py

```diff
@@ -20,6 +25,23 @@ def student_graph(student_ids):
     # Get the scores needed
     assert(len(dbData) != 0), "Cannot find student data."
 
+
+    # create summary graph if > 6 students
+    # else:
+        # find avg val for each score
+    avgScores = [float(sum(col))/len(col) for col in zip(*scoreSet)]
+    plt.bar(yPos, avgScores, align = 'center', color='#3fe0d0', alpha = 0.5, label = 'Average Values')
+
+    # Make graph
+    plt.xticks(yPos, labels)
+    plt.ylabel("Score")
+    plt.title("Average Student Distribution of Types")
+    plt.legend( bbox_to_anchor=(1.05, 1), loc=2, borderaxespad=0. )
+
+    name = 'app/assets/images/summary.png'
+    plt.savefig(name, bbox_inches='tight')
+    plt.cla()
+
```

Comment:
I think that the code would be cleaner if this was a separate function. Same with the graph section below.